primitives/container-chain-genesis-data/src/lib.rs

```diff
@@ -97,6 +98,7 @@ impl Default for TokenMetadata {
             token_symbol: BoundedVec::truncate_from(b"UNIT".to_vec()),
             ss58_format: 42,
             token_decimals: 12,
+            is_ethereum: false,
```

Comment:
Since this is not token metadata per se, I would make a new struct of the form:

```
struct Properties {
token_metadata: TokenMetadata
is_ethereum: bool
}
```